__tests__/errorsHandled.test.js

```diff
@@ -37,8 +37,43 @@ describe("GET /api/:invalid endpoint", ()=>{
         .get('/api/invalid')
         .expect(404) 
         .then((res)=>{
-            //console.log(Object.keys(res));
             expect(res.body.msg).toBe("Invalid endpoint");
         })
     })
+
+})
+
+describe("GET /api/reviews/:review_id", ()=>{
+    test("GET /api/reviews responds with a review object, which has the following keys: review_id, title, review_body, designer, review_img_url, votes, category, owner, create_at", ()=>{
+        return request(app)
+        .get('/api/reviews/1')
+        .expect(200)
+        .then((res)=>{
+            expect(res.body[0]).toEqual(expect.objectContaining({
+                review_id:1,
+                title:'Agricola',
+                review_body:'Farmyard fun!',
+                designer: 'Uwe Rosenberg',
+                review_img_url:'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                votes: 1,
+                category:'euro game',
+                owner:'mallionaire',
+                created_at: '2021-01-18T10:00:20.514Z',
+            }))
+        })
+    })
+})
+
+describe.only("GET /api/reviews/:invalid_endpoint", ()=>{
+    test("GET /api/reviews/666", ()=>{
+        return request(app)
+        .get(`/api/reviews/666`)
+        .expect(404)
```

Comment:
I can see that you are testing for the error code but we need to test for the error message